spec/requests/new_survey_response_spec.rb

```diff
@@ -35,7 +35,8 @@
       ignored:      "ignore me",
       email:        "omar@sumofus.org",
       source:       'FB',
-      akid:         '3.4234.fsdf'
+      akid:         '3.4234.fsdf',
+      fields:       {}
```

Comment:
The spec was failing without the addition of this key because it was referencing this field. Not sure why it didn't fail before.